contributing/community/speaker-mentoring.rst

```diff
@@ -0,0 +1,43 @@
+Speaker mentoring
+=================
+
+The Symfony community benefits greatly from as many people as possible
+sharing their knowledge and experience with others. Every different
+point of view adds to our collective understanding of how to best use
+and evolve the code, design patterns and architecture provided within
+the Symfony community. As such we specifically welcome not only sharing
+from long time contributors, but also from new users as they will come
+across entirely different challenges with a totally fresh new look and
+perspective.
+
+How to get started
+------------------
+
+Given a first talk at a conference can seem quite intimidating but we
+want to make sure that as many people as possible are empowered to take
+this path if they are motivated. Therefore we have collected a few
+resources with advice. More importantly we connect experienced speakers
```

Comment:
> Therefore we have collected a few resources with advice

This sentence sounds a bit weird… to me. Maybe check with a native english speaker?